sdk/pom.xml

```diff
@@ -269,6 +269,11 @@
             <artifactId>jackson-databind</artifactId>
             <version>${fasterxml-version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>jsr305</artifactId>
+            <version>3.0.2</version>
+        </dependency>
```

Comment:
What is this for?

Reply:
new dependency in the latest version of the generator, for `@Nullable` annotations